lib/engine/step/g_1889/special_track.rb

```diff
@@ -19,7 +19,7 @@ def blocking?
         def process_lay_tile(action)
           return super unless action.entity == @company
 
-          lay_tile(action, spender: @round.company_seller)
+          lay_tile(action, spender: @round.company_sellers.any? ? @round.company_sellers.first : nil)
```

Comment:
```suggestion
          lay_tile(action, spender: @round.company_sellers.first)
```

`[].first` is `nil`, so this code should have the same effect

Reply:
Fixed

Reply:
`@round.company_sellers.any? ? @round.company_sellers.first` is a broken ternary (which broke the build) rather than just `@round.company_sellers.first`